ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -365,4 +358,68 @@ open class SideTabBar: UIView {
             delegate?.sideTabBar?(self, didSelect: item, fromTop: false)
         }
     }
+
+    public func overrideTokens(_ tokens: SideTabBarTokens?) -> Self {
+        overrideTokens = tokens
+        return self
+    }
+
+    var defaultTokens: SideTabBarTokens = .init()
+    var tokens: SideTabBarTokens = .init()
+    var overrideTokens: SideTabBarTokens? {
+        didSet {
+            updateSideTabBarTokens()
+        }
+    }
+
+    private class CustomSideTabBarItemTokens: TabBarItemTokens {
```

Comment:
No, because we are assigning these tokens to the TabBarItems, not SideTabBar. They use the super override tokens coming from SideTabBarTokens, but need to be of type TabBarItemTokens to be able to be set on the TabBarItems.